include/ck/utility/type_convert.hpp

```diff
@@ -100,6 +100,15 @@ template <>
 inline __host__ __device__ f8_t type_convert<f8_t, float>(float x)
 {
 #if defined(__gfx940__) || defined(__gfx941__) || defined(__gfx942__)
+    float max_fp8 = 240.0f;
+    if(x > max_fp8)
```

Comment:
Could you change the if-else branch as below to avoid branches in a GPU kernel.

`x = x > max_fp8 ? max_fp8 : (x < -max_fp8 ? -max_fp8 : x);`